Makefile

```diff
@@ -23,13 +23,13 @@ lint: $(GOLANGCI_LINT)
 test: test-go
 test-go:
 	@ echo; echo "### $@:"
-	GO111MODULE=on go test ./pkg/...
+	@ echo "No need to run test. Testing is done in GitHub Actions and custom Prow job."
```

Comment:
I think this is ok and not worth it to set additional infra for this to work given that we already have other jobs that are running the tests we want:

- integration tests are run in Prow (pull-kubernetes-csi-csi-proxy-integration)
- unit tests are run in Windows Tests / unit_tests (1.16, windows-latest) (pull_request)